lib/m3u8/reader.rb

```diff
@@ -145,6 +148,14 @@ def parse_stream_attributes(attributes)
       end
     end
 
+    def parse_segment_discontinuity_tag(line)
```

Comment:
Unused method argument - `line`. If it's necessary, use `_` or `_line` as an argument name to indicate that it won't be used. You can also write as `parse_segment_discontinuity_tag(*)` if you want the method to accept any arguments but don't care about them.